chat-widget/src/components/livechatwidget/common/setPostChatContextAndLoadSurvey.ts

```diff
@@ -8,7 +8,7 @@ import { LiveChatWidgetActionType } from "../../../contexts/common/LiveChatWidge
 import { TelemetryHelper } from "../../../common/telemetry/TelemetryHelper";
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const setPostChatContextAndLoadSurvey = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAction>, persistedChat?: boolean,) => {
+export const setPostChatContextAndLoadSurvey = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAction>, persistedChat?: boolean, isPostchatEnabled?:boolean) => {
     try {
```

Comment:
Would it be simpler if just break the code execution as soon as the feature is disabled?

```
if (isPostchatEnabled()) { return; }
```